requirements.txt

```diff
@@ -4,13 +4,17 @@ colorama==0.4.6
 greenlet==3.0.3
 idna==3.7
 iniconfig==2.0.0
+Jinja2==3.1.4
+MarkupSafe==2.1.5
 packaging==24.1
 playwright==1.44.0
 pluggy==1.5.0
 psutil==5.9.8
 pyee==11.1.0
 pytest==8.2.2
 pytest-base-url==2.1.0
+pytest-html==4.1.1
```

Comment:
[pytest-html](https://pypi.org/project/pytest-html/) is the main dependency. The other 3 packages added are transient dependencies.

Reply:
As discussed yesterday, I think this is fine given Python is only needed for tests - we probably don't need to go for something like Poetry at this point.